pkg/source/adapter/adapter.go

```diff
@@ -95,14 +103,27 @@ func (a *Adapter) Start(ctx context.Context) error {
 		zap.String("Namespace", a.config.Namespace),
 	)
 
+	// Init control service
+	a.controlServer, err = ctrlnetwork.StartInsecureControlServer(ctx)
```

Comment:
insecrure :-)

Reply:
yeah for this first pass, we don't encrypt the connection with mTLS, I think the certificates controller still need some work.